fiware-region-sanity-tests/commons/http_phonehome_server.py

```diff
@@ -113,8 +115,19 @@ def before_request_body():
     """
     Add a Tool to our new Toolbox.
     """
-    transaction_id = "txid:" + cherrypy.request.headers['TransactionId']
+    logger.info("before_request_body: %s ", cherrypy.request.params)
+    if 'TransactionId' in cherrypy.request.headers:
+        transaction_id = cherrypy.request.headers['TransactionId']
+    elif 'TransactionId' in cherrypy.request.params:
+        transaction_id = cherrypy.request.params['TransactionId']
+        cherrypy.request.params = {}
+    else:
+        transaction_id = str(uuid.uuid1())
+
+    cherrypy.request.transaction_id = transaction_id
+    transaction_id = "txid:" + transaction_id
```

Comment:
because I prefer to save/load only the id in "cherrypy.request" instead of the string with "txid:"